lib/generator.rb

```diff
@@ -3,6 +3,7 @@
 require 'erb'
 require 'json'
 require 'ostruct'
+require 'fileutils'
 
```

Comment:
I don't think we should shy away from using stdlib. It allows us to be more targeted and is only a `require` away.